src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Event.cs

```diff
@@ -72,13 +75,13 @@ private enum EventAssignmentKind
         /// <remarks>
         /// TODO: use or delete isDynamic.
         /// </remarks>
-        private BoundExpression RewriteWindowsRuntimeEventAssignmentOperator(SyntaxNode syntax, EventSymbol eventSymbol, EventAssignmentKind kind, bool isDynamic, BoundExpression rewrittenReceiverOpt, BoundExpression rewrittenArgument)
+        private BoundExpression RewriteWindowsRuntimeEventAssignmentOperator(SyntaxNode syntax, EventSymbol eventSymbol, EventAssignmentKind kind, bool isDynamic, BoundExpression? rewrittenReceiverOpt, BoundExpression rewrittenArgument)
         {
-            BoundAssignmentOperator tempAssignment = null;
-            BoundLocal boundTemp = null;
-            if (!eventSymbol.IsStatic && CanChangeValueBetweenReads(rewrittenReceiverOpt))
+            BoundAssignmentOperator? tempAssignment = null;
+            BoundLocal? boundTemp = null;
+            if (!eventSymbol.IsStatic && CanChangeValueBetweenReads(rewrittenReceiverOpt!))
```

Comment:
I didn't understand how we justify these suppressions

Reply:
Please add an assert for `eventSymbol.IsStatic || rewrittenReceiverOpt is {}`